cfpb/cfgov-refresh
# Wagtail: Add notification molecule to sublanding pages

## Changes

- Update `notification.html` to support `value` passed in from wagtail.

## Testing

1. Create a new sublanding page in wagtail and add a notification to the content area. ([Shortcut](http://localhost:8000/admin/pages/add/v1/sublandingpage/319/))
2. Set the "message" only and preview the page to see that the notification type is the default and the message shows up.
3. Change the type drop-down and preview the page to see the color and icon change in the notification.
4. Add the explanation text and preview the page to see that it is added below the notification message.
5. Add links and preview the page to see that links appear after the explanation text.
5. See that hardcoded notifications on e.g. http://localhost:8000/about-us/blog/ still work (e.g. enter date before 1900).

## Screenshots

NEW SCREENSHOTS (4/5)

<img width="1194" alt="screen shot 2019-03-05 at 11 46 21 am" src="https://user-images.githubusercontent.com/704760/53821704-50575880-3f3c-11e9-94f2-ece60266a442.png">

<img width="611" alt="screen shot 2019-03-05 at 10 33 07 am" src="https://user-images.githubusercontent.com/704760/53821663-3fa6e280-3f3c-11e9-884f-09797ed59497.png">

<img width="367" alt="screen shot 2019-03-05 at 10 33 17 am" src="https://user-images.githubusercontent.com/704760/53821666-42093c80-3f3c-11e9-9190-cba5c266acc3.png">
